app/main/helpers/__init__.py

```diff
@@ -2,6 +2,8 @@
 from datetime import datetime
 from functools import wraps
 import hashlib
+import os
```

Comment:
Do we need these imports in here when the rest of the file hasn't changed?

Reply:
Nope, they were leftovers after a refactor, I'll whip them out. I'm surprised flake-8 did't catch them though.

Reply:
These could still be removed.